cmd/validate/image.go

```diff
@@ -249,7 +249,15 @@ func validateImageCmd(validate imageValidationFunc) *cobra.Command {
 							if len(parts) < 2 {
 								log.Errorf("Incorrect syntax for --extra-rule-data")
 							}
-							unmarshaled[parts[0]] = parts[1]
+							if source_utils.SourceIsFile(parts[1]) && utils.HasJsonOrYamlExt(parts[1]) {
```

Comment:
To me it looks like we would overwrite any `sources[i].RuleData` that was already set on the policy here with `--extra-rule-data` flag (that might be a bug prior to this change). I don't see any prior test coverage for when both `ruleData` in policy and `--extra-rule-data` is provided.

Reply:
I can confirm it wasn't a bug. I added `ruleData` to my unit test along with `--extra-rule-data` and it seemed to work fine. Please let me know what you think